Source/Core/VideoBackends/D3D/D3DBase.cpp

```diff
@@ -324,45 +293,34 @@ HRESULT Create(HWND wnd)
     UpdateActiveConfig();
   }
 
-  DXGI_SWAP_CHAIN_DESC swap_chain_desc = {};
-  swap_chain_desc.BufferCount = 1;
+  DXGI_SWAP_CHAIN_DESC1 swap_chain_desc = {};
+  swap_chain_desc.BufferCount = NUM_SWAPCHAIN_BUFFERS;
```

Comment:
This is an implicit change for a flipping swap-chain which is required for Quad-Buffered stereoscopy. Setting the BufferCount to 1 actually means we were doing front-buffered rendering in exclusive fullscreen:

> When you call IDXGIFactory::CreateSwapChain to create a full-screen swap chain, you typically include the front buffer in this value.

> In full-screen mode, there is a dedicated front buffer; in windowed mode, the desktop is the front buffer.